cmake/switch/Switch.cmake

```diff
@@ -81,16 +81,22 @@ function(nx_generate_nacp outfile)
 
 	add_custom_command(
 		OUTPUT "${outfile}"
-		COMMAND "${NX_NACPTOOL_EXE}" --create "${NACP_NAME}" "${NACP_AUTHOR}" "${NACP_VERSION}" "${outfile}"
+		COMMAND "${NX_NACPTOOL_EXE}" --create "$<TARGET_FILE:${NACP_NAME}>" "${NACP_AUTHOR}" "${NACP_VERSION}" "${outfile}"
+		DEPENDS ${NACP_NAME}
```

Comment:
Why was this changed? nx_generate_nacp isn't intended to depend on any target.

Reply:
Oops, this is a leftover of my hacking to fix this + my lack of understanding the nx toolchain.
I'll remove this change.

Reply:
Oops, this is a leftover of my hacking to fix this + my lack of understanding the nx toolchain.
I'll remove this change.

Reply:
Removed.